src/pybind/cephfs.py

```diff
@@ -299,6 +307,40 @@ def chdir(self, path):
         if ret < 0:
             raise make_ex(ret, "chdir failed")
 
+    def isdir(self, dirent):
```

Comment:
Rather than hardcoding 0x4 here, we should expose the various DT_* constants up to the user.  They might also like to check whether a dentry is a link, etc.